velox/exec/fuzzer/PrestoSql.cpp

```diff
@@ -270,6 +270,22 @@ std::string toCallSql(const core::CallTypedExprPtr& call) {
     sql << "[";
     toCallInputsSql({call->inputs()[1]}, sql);
     sql << "]";
+  } else if (call->name() == "switch") {
+    const auto& inputs = call->inputs();
+    sql << " case ";
+    size_t inputSize = inputs.size();
+    size_t inputIndex = 0;
+    while (inputIndex < inputSize - 1) {
+      sql << " when ";
+      toCallInputsSql({inputs[inputIndex++]}, sql);
+      sql << " then ";
+      toCallInputsSql({inputs[inputIndex++]}, sql);
+    }
+    if (inputIndex < inputSize) {
+      sql << " else ";
+      toCallInputsSql({inputs[inputIndex]}, sql);
+    }
+    sql << " end ";
```

Comment:
Please add a unit test in PrestoSqlTest.cpp. By the way, I recall that Peter is also fixing the sql translation for switch. Could you reach out to him to avoid duplicate effort (or borrow his unit test)?

Reply:
Looks like I can just delete this, someone else already added it.